app/controllers/concerns/filter_parameters.rb

```diff
@@ -22,9 +22,12 @@ def merge_previous_parameters(all_parameters)
   end
 
   def parameters
-    return request.query_parameters if %w(GET HEAD).include?(request.method)
-
-    request.request_parameters
+    request_params = if %w(GET HEAD).include?(request.method)
+                       request.query_parameters
+                     else
+                       request.request_parameters
+                     end
+    request_params.except("page") # exclude pagination params
```

Comment:
Could we do this in the `filter_params` method where we're already filtering out some unwanted parameters?